### ISSUE TITLE
How can I hide the negative values from a vertical axis?

### ISSUE THREAD

**Issue by ArchieCoder** [ScottPlot/ScottPlot#4272]
**Question:**

I have a energy plot (amperage, voltage, power). The concept of negative energy does not apply to my scenario. Is there a way to show the plot starting at 0?

**ScottPlot Version:**
5.0.39

![image](https://github.com/user-attachments/assets/20931cb2-0358-40e0-8df2-d569b41904e6)

**Comment by CoderPM2011** [comment]
Look at this.
use `SetLimits`
https://scottplot.net/cookbook/5.0/AxisAndTicks/SetAxisLimits/
or use `SetLimitsY`
https://scottplot.net/cookbook/5.0/AdvancedAxis/InvertedAxis/

**Comment by ArchieCoder** [comment]
@CoderPM2011 Thanks for your answer. SetLimitsY does not do anything. Am I doing something wrong?

![image](https://github.com/user-attachments/assets/e8f72365-59dd-4029-be0b-dfd7c111800e)

**Comment by CoderPM2011** [comment]
@ArchieCoder 
Maybe it has something to do with you having more than one axis?
Can you try using SetLimitsY before finally displaying the data?

**Comment by ArchieCoder** [comment]
@CoderPM2011 Even if I have one vertical axis and set the limit before the data it does not work

![image](https://github.com/user-attachments/assets/ef5051da-5092-4141-9947-ed31aa746540)

**Comment by CoderPM2011** [comment]
@ArchieCoder 
Sorry, I didn't express myself clearly.😅

Use the `SetLimitsY` method only after setting the `YAxis`, as I suspect `SetLimitsY` is not applied to the plot's `YAxis`.

**Comment by ArchieCoder** [comment]
@CoderPM2011 I simplified the whole thing to do as the example. It does not work :/

![image](https://github.com/user-attachments/assets/fa1643f4-2535-4d52-90e3-c93ae0d780ec)

**Comment by CoderPM2011** [comment]
@ArchieCoder maybe try this.
`Plot.Axes.Margins(0, 0);`

**Comment by ArchieCoder** [comment]
@CoderPM2011 Margins(0, 0) does something! At least, it makes the Y axis starting at 0. Thank you for that.

**Comment by swharden** [comment]
Note that there are "also rules" that could be used to prevent users from dragging the plot in the negative area

https://scottplot.net/demo/5.0/#axis-rules

**Comment by ArchieCoder** [comment]
@swharden When multiple Y-axis are used, the lock vertical feature breaks the chart.

1- Try the code from the sample
2- Comment line 24 and run again

[ScottPlotSample.zip](https://github.com/user-attachments/files/17950070/ScottPlotSample.zip)

**Comment by swharden** [comment]
> @swharden When multiple Y-axis are used, the lock vertical feature breaks the chart.
> 
> 1- Try the code from the sample
> 2- Comment line 24 and run again
> 
> [ScottPlotSample.zip](https://github.com/user-attachments/files/17950070/ScottPlotSample.zip)
> 

Hi @ArchieCoder, I am on mobile so I will be brief,

If you can share the minimal source code required to reproduce this issue as text here, it will increase the likelihood of a helpful response in less time.

If not, I can take a closer look when there's more time and when I'm on a workstation computer again, but it's not easy for me to predict when that will be

**Comment by swharden** [comment]
Quick follow-up, I just noticed this comment is on a closed issue. If there is an issue that remains in the present version of this package, I recommend you open a new issue and we can take a closer look!

**Comment by swharden** [comment]
Quick follow-up, I just noticed this comment is on a closed issue. If there is an issue that remains in the present version of this package, I recommend you open a new issue and we can take a closer look!

**Comment by ArchieCoder** [comment]
> e

Created this https://github.com/ScottPlot/ScottPlot/issues/4581
